jxl_cli/src/main.rs

```diff
@@ -27,6 +30,13 @@ fn parse_jxl_codestream(data: &[u8]) -> Result<(), jxl::error::Error> {
         let r = read_icc(&mut br)?;
         println!("found {}-byte ICC", r.len());
```

Comment:
not super related, but I suppose we can at some point not print so much information here, but rather in jxl-inspect instead.

Reply:
when we add clap for CLI parsing, we can have a --quiet flag to control these as in libjxl